package.json

```diff
@@ -77,13 +78,14 @@
     "@scripts/build": "workspace:*",
     "@types/react": "^18",
     "@types/react-dom": "^18",
-    "@vitest/ui": "^0.33.0",
     "@vitest/coverage-v8": "^0.33.0",
+    "@vitest/ui": "^0.33.0",
     "esbuild": "0.17.19",
     "eslint": "^8.28.0",
-    "lint-staged": "~13.1.0",
     "husky": "^8.0.0",
-    "turbo": "1.10.2",
+    "lint-staged": "~13.1.0",
+    "nx": "16.10.0",
+    "nx-cloud": "^16.5.2",
```

Comment:
if we do not use `nx-cloud`, can we remove it from dependencies? this can reduce the dependencies number.

Reply:
It has indeed been used.